src/app/_utils/getCMSOptionsWithDefault.ts

```diff
@@ -0,0 +1,23 @@
+import type { DefaultFilterOptionType } from '@/types/filterTypes'
+
+import {
+  getCMSFieldOptionsAndValidIds,
+  type CMSFieldOptionsAndValidIdsParams,
+} from './getCMSFieldOptionsAndValidIds'
+
+type TODO = CMSFieldOptionsAndValidIdsParams & {
```

Comment:
`TODO`?

Reply:
Is there a reason for not merging this with `getCMSFieldOptionsAndValidIds`?

Reply:
🙃